src/app/common/components/dataset-kind/dataset-kind.component.html

```diff
@@ -0,0 +1,9 @@
+<span
+    [ngClass]="{
+        root: isRoot,
+        derivative: !isRoot,
+    }"
+    data-test-id="dataset-kind"
+    class="label label--secondary v-align-middle"
+    >{{ isRoot ? "Root" : "Derivative" }}</span
+>
```

Comment:
It does not look like this piece has enough responsibilities or reuse potential to really be a separate component. Do we plan to use it anywhere else?